### DISCUSSION TITLE
Requires the 2015 Audi TT firmware version MU0036 MIB patch

### DISCUSSION THREAD

**Discussion by jeweitian** [Mr-MIBonk/M.I.B._More-Incredible-Bash#199] [Q&A]
![微信图片_20230506203252](https://user-images.githubusercontent.com/127286399/236626381-31f90f4e-8a67-46f0-9488-54f0fb911795.jpg)

**Comment by jeweitian** [comment]
I want to activate the unit carplay function

**Reply by harman-f** [reply]
M.I.B does most likely not support this old FW version. 
try to create a backup.

**Reply by jeweitian** [reply]
The firmware MU0036 backup of the 2015 Audi TT has been uploaded to MIB/Backup
